pyloras/_loras.py

```diff
@@ -128,7 +129,7 @@ def _initialize_params(self, X, y, rng):
 
         if self.manifold_learner:
             self._check_2d_manifold_learner()
-            self.manifold_learner_ = self.manifold_learner
```

Comment:
This looks good @vortex-17 , thank you for taking the time to contribute. I think we could now just return the cloned manifold_learner using the `_check_2d_manifold_learner` method to make things look a little cleaner and have a single line like:
```python
self.manifold_learner_ = self._check_2d_manifold_learner()
```